src/main/java/com/chekanova/imagetool/service/comparison/impl/DrawSquareDifferenceServiceImpl.java

```diff
@@ -75,21 +76,22 @@ private Box buildBox(int x, int y, int w, int h, boolean[][] comparison) {
         return box;
     }
 
-    private void drawBox(BufferedImage image, Box box) {
+    private void drawBox(BufferedImage image, Box box, String frameColor) {
         if (box == null) {
             return;
         }
+        int rgb = Color.decode(frameColor).getRGB();
```

Comment:
Please move this logic to src/main/java/com/chekanova/imagetool/service/impl/ImageServiceImpl.java

Reply:
Done. Also refactored test class